db/migrate/20240205141443_create_authentication_tokens.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateAuthenticationTokens < ActiveRecord::Migration[7.1]
```

Comment:
It would be good to have the migrations in a separate PR that can be deployed before the code that depends on the schema changes. It probably won't matter in this case because the dependencies are all feature flagged.